java/developer-guide/knowledge-base/technical-articles/important-icalendar-rfc-2445-details/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Important iCalendar RFC 2445 Details
+type: docs
+weight: 70
+url: /java/important-icalendar-rfc-2445-details/
+---
+
+
+## **About Aspose iCalendar Object Model**
+The Aspose.Email contains all the classes provided by the Aspose iCalendar component RecurrencePattern and RecurrenceRule are the central classes of Aspose iCalendar, and provide concrete implementations of the corresponding RFC 2445 elements. 
```

Comment:
Add links to  [RecurrencePattern](https://apireference.aspose.com/email/java/com.aspose.email/RecurrenceRule), [RecurrenceRule](https://apireference.aspose.com/email/java/com.aspose.email/RecurrenceRule) and [iCalendar](https://apireference.aspose.com/email/java/com.aspose.email/MapiCalendar)